pages/dkp/kaptain/1.3.0/install/air-gapped-konvoy/index.md

```diff
@@ -26,8 +26,10 @@ Use the following instructions to install Spark Operator from Kommander Catalog
 The installation steps for Kaptain on an air-gapped cluster are as follows:
 
 * Download `kaptain_air_gapped.tar.gz` that will contain the required artifacts to perform an air-gapped installation.
+<!-- NEW TAR because of Minio -->
```

Comment:
Is this needed now?

Reply:
This one will remain the same - No additional untar is needed as the notice file is just added a the root.

Reply:
```suggestion
```